views/matrixsplit.html

```diff
@@ -233,8 +233,8 @@
             =
             \begin{bmatrix}
```

Comment:
I see, thanks.

Would it be fine for you to show the non-simplified form and then the simplified form directly in terms of Bernstein's polynomials? Something like this: http://mathb.in/42121

I say this because I don't have Mathematica, and I suspect many others don't as well. It's pretty straightforward for anyone to understand how the non-simplified form comes out, and to later understand the transformation to the Bernstein form from there.